website/public/js/controllers/inventoryCtrl.js

```diff
@@ -215,23 +215,36 @@ habitrpg.controller("InventoryCtrl",
       $rootScope.selectedQuest = item;
       $rootScope.openModal('buyQuest', {controller:'InventoryCtrl'});
     }
-    
+
     $scope.getSeasonalShopArray = function(set){
       var flatGearArray = _.toArray(Content.gear.flat);
-      
+
       var filteredArray = _.where(flatGearArray, {index: set});
 
       return filteredArray;
     };
-    
+
     $scope.getSeasonalShopQuests = function(set){
       var questArray = _.toArray(Content.quests);
-      
+
       var filteredArray = _.filter(questArray, function(q){
         return q.key == "evilsanta" || q.key == "evilsanta2";
       });
 
       return filteredArray;
     };
+
+    $scope.dequip = function(){
+
+      for ( item in user.items.gear.equipped ){
+        var itemKey = user.items.gear.equipped[item];
+        if ( user.items.gear.owned[itemKey] ) {
```

Comment:
This `if` statement isn't really necessary - if the player has equipped the  item, they would **always** want it dequipped if they click the button, so there's no need for an `if`. In some unusual cases it would actually be undesirable to have the `if` there - it is occasionally possible for the avatar to be equipped with items that the user does not own (e.g., https://github.com/HabitRPG/habitrpg/issues/3729).